### ISSUE TITLE
Support 21-23% quality gems

### ISSUE THREAD

**Issue by fqrwx** [Stickymaddness/Procurement#929]
When I set price to one of these gems, they are not included in the forum code, I had to manually add them

**Comment by thailyn** [comment]
Under which category are they not showing up/should they show up in?

**Comment by fqrwx** [comment]
I believe they need to be supported in the template like this one but for 21 22 and 23:

```
[spoiler="          Quality Gems 15-20    "]
{Quality16Gems}{Quality17Gems}{Quality18Gems}{Quality19Gems}{Quality20Gems}
[/spoiler]
```

**Comment by fqrwx** [comment]
found some minor issue with the gem spoiler titles too

5-10, 10-15, 15-20 needs to be replaced with 6-10, 11-15 and 16-20 respectively

**Comment by fqrwx** [comment]
is there a {Tier16Maps} needed for tier 16 maps also?

**Comment by aydjay** [comment]
Closed by #932
